src/less/workingfiles/submit.less

```diff
@@ -140,7 +140,7 @@ input[type="radio"] {display:none;}
 .import-serp-select-wrapper {
     display: flex;
     justify-content: space-between;
-    width: 350px;
+    width: 250px;
```

Comment:
A tip is to use `max-width` instead of `width` whenever you can. That way you get nice reflow when the window size is changed